data/competencies.yml

```diff
@@ -1,16 +1,292 @@
+  level: junior
+  area: technical
+  domain: null
+
+- id: 1ab2d281-1a60-4a49-a1d3-87f03bcf510e
+  summary: Uses version control to manage routine development workflow, and can work with support on uncommon workflows
+  examples:
+    - "Git example: is able to clone a repo, check out a branch, add, commit, push"
+    - "Git example: asks for help with more complex tasks like rebase and resolving conflicts"
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: 43cd4e9d-6462-4d4b-94e2-69dfeff246f6
+  summary: Uses code to make something reasonably straightforward, mostly independently
+  examples:
+    - Works on a feature or improves an existing one
+    - Adds a use case to an existing automation script
+    - Writes a CloudFormation template
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: 655e2933-b58e-488b-9683-a0b191a37788
+  summary: Writes automated unit or end to end tests with help from a more senior engineer
+  examples: []
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: cb1a4c9c-28dd-4d1d-938e-1f689fb91393
+  summary: Fixes or updates tests when prompted
+  examples: []
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: 43f6e213-b51e-45bd-bb37-a23a3e7d5457
+  summary: Checks whether code can be reused before writing new code, and generally identifies reuse when there is the opportunity
+  examples:
+    - Checks to use a Python/Node/etc package in project
+    - Checks for the chance to integrate with proprietary software packages
+    - Checks Origami capabilities
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: cad93c51-dee9-4b10-8ccb-d2936dd95380
+  summary: Works to timeboxes when debugging
+  examples:
+    - Ensures they have a timebox when debugging
+    - Communicates their progress through a timebox
+    - Seeks help from other engineers before time runs out
+    - Does not overshoot a timebox without flagging it
+  supportingUrls: []
+  description: null
+  level: junior
   area: technical
   domain: null
 
+- id: 0137fb68-807a-4971-8431-1b8bfd9a4bae
+  summary: Shadows during live incidents or joins debugging mob sessions
+  examples: []
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: 9098cc7f-a50a-48df-b84f-c4ee9fa47dbe
+  summary: Is able to explain why using CI/CD pipelines for automation is beneficial, and uses them for basic functions
+  examples:
+    - Restarts a broken build
+    - Is able to interpret CI output and articulate what is failing
+    - Can fix simple problems or asks for help
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: ae32dc05-3e53-47d2-a027-345eb42dffa3
+  summary: Occasionally uses monitoring tools to identify problems
```

Comment:
Not sure about this being a requirement for a junior engineer given that they are only expected to shadow on incidents.